src/main/java/com/microsoft/azure/spring/data/cosmosdb/repository/support/DocumentDbEntityInformation.java

```diff
@@ -91,7 +86,7 @@ public String getPartitionKeyFieldValue(T entity) {
         return partitionKeyField == null ? null : (String) ReflectionUtils.getField(partitionKeyField, entity);
     }
 
-    public boolean isIdFieldAsPartitonKey() {
+    public boolean isIdFieldAsPartitionKey() {
         final String keyName = getPartitionKeyFieldName();
```

Comment:
This `isIdFieldAsPartitionKey` will not be used in this class?

Reply:
yes, this is for corner case, partitionkey act on Id Field.